amundsen_application/static/jest.config.js

```diff
@@ -45,6 +44,9 @@ module.exports = {
     'jsx',
     'json',
   ],
+  moduleNameMapper: {
+    '^.+\\.(css|scss)$': '<rootDir>/node_modules/jest-css-modules',
+  },
```

Comment:
According to `jest-css-modules` documentation, we're supposed to use `moduleNameMapper`. I'm not sure how or why the rule in `transform` rule was working before.

Reply:
Beats me. This config was the result of reading through various documentation, github issues, and stackoverflow threads. I wonder if its as simple as this being the way to do things with the newer versions. Either way, looks good to me.